components/atoms/forms/FileInput/FileInput.js

```diff
@@ -38,6 +38,73 @@ export default function FileInput({
   const [hasFocus, setHasFocus] = useState(false);
   const fileInputID = useMemo(() => id || uniqueId("fileInput_"), [id]);
 
+  // const updateFiles = useCallback(
```

Comment:
remove commented code

Reply:
will work on it later have to continue some doubt to be resolved